src/Bellhop.js

```diff
@@ -307,7 +309,7 @@ export class Bellhop extends BellhopEventDispatcher {
    */
   logDebugMessage(received = false, message) {
     if (this.debug && typeof this.debug === 'function') {
-      this.debug({ isChild: this.isChild, received: false, message: message });
+      this.debug({ isChild: this.isChild, received, message: message });
```

Comment:
@902seanryan @deycorinne Please check this line also. 
`false` is suppsed to be removed.
thank you.